docs/package.md

```diff
@@ -272,15 +283,15 @@ They can also use functions from the [install library](scripts/install-lib).
 When installing a new package, the following happens:
 
 * The package files are unpacked (but not installed)
-* `preinstall` is called, if it exists
+* `preinstall` is called if it exists
```

Comment:
"if it exists, preinstall is called" and "preinstall is called if it exists" are valid, i think. something something dependent / independent clauses and commas

Reply:
True, I can just see us flip-flopping back and forth on this one depending on who's submitting the PR.

Reply:
I’m not sure about this since I’m not a native speaker. [This source](https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html) says “don't put a comma after the main clause when a dependent (subordinate) clause follows it (except for cases of extreme contrast).” It looks like this rule applies here, doesn’t it?

Reply:
I would argue that this is a case of extreme contrast. That said English grammar is highly subjective and changes constantly.